Server/.editorconfig

```diff
@@ -0,0 +1,234 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space

```

Comment:
Indenting met spaties 😅

Reply:
Klopt deze blijven in alle ontwikkel omgevingen gelijk.